drivers/overlay/overlayutils/utils.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 var (
+	mutex        sync.RWMutex
```

Comment:
to explain this change; it's convention to put the mutex above the variables that it's protecting